permissions/plugin-kryptowire.yml

```diff
@@ -8,3 +8,4 @@ developers:
 - "mnairn"
```

Comment:
ok, `lrosset` on https://issues.jenkins-ci.org/secure/ViewProfile.jspa?name=lrossett `==` @odra 

We can move forward then.